www/src/components/sidebar/section.js

```diff
@@ -0,0 +1,249 @@
+import React from "react"
+
+import getActiveItem from "../../utils/sidebar/get-active-item"
+import presets, { colors } from "../../utils/presets"
+import { scale, rhythm } from "../../utils/typography"
+
+import SectionTitle from "./section-title"
+import ChevronSvg from "./chevron-svg"
+
+const horizontalPadding = rhythm(3 / 4)
+const horizonalPaddingDesktop = rhythm(3 / 2)
+
+const ToggleSectionButton = ({
+  title,
+  isActive,
+  uid,
+  hideSectionTitle,
+  onSectionTitleClick,
+}) => (
+  <button
+    aria-expanded={isActive}
+    aria-controls={uid}
+    css={{
+      ...styles.button,
+      [presets.Tablet]: {
+        display: hideSectionTitle ? `none` : `inline`,
+      },
+    }}
+    onClick={onSectionTitleClick}
+  >
+    <SectionTitle isActive={isActive}>
+      {title}
+      <ChevronSvg
+        cssProps={{
+          color: colors.gray.light,
+          marginLeft: `auto`,
+          transform: isActive ? `rotateX(180deg)` : `rotateX(0deg)`,
+          transition: `transform 0.2s ease`,
+          display: `none`,
+          [presets.Tablet]: {
+            display: `inline-block`,
+          },
+        }}
+      />
+    </SectionTitle>
+  </button>
+)
+
+const Title = ({ title }) => (
+  <div
+    css={{
+      paddingLeft: horizontalPadding,
+      paddingRight: horizontalPadding,
+      [presets.Desktop]: {
+        paddingLeft: horizonalPaddingDesktop,
+        paddingRight: horizonalPaddingDesktop,
+      },
+    }}
+  >
+    <SectionTitle>{title}</SectionTitle>
+  </div>
+)
+
+class Section extends React.Component {
+  state = { uid: (`` + Math.random()).replace(/\D/g, ``) }
+
+  _isChildItemActive = (item, activeItemId) => {
+    if (item.subitems) {
+      const matches = item.subitems.filter(function(subitem) {
+        return subitem.link === activeItemId && item.link === subitem.parentLink
+      })
+
+      return matches.length >= 1
+    }
+  }
+
+  render() {
+    const {
+      activeItemHash,
+      createLink,
+      location,
+      onLinkClick,
+      onSectionTitleClick,
+      section,
+      hideSectionTitle,
+      singleSection,
+    } = this.props
+    const uid = `section_` + this.state.uid
+    const activeItemId = getActiveItem(section, location, activeItemHash)
+    const SectionTitleComponent = section.disableAccordions
+      ? Title
+      : ToggleSectionButton
+
+    let { isActive } = this.props
+    if (section.disableAccordions) {
+      isActive = true
+    }
```

Comment:
don't modify props -- do something like:

`const isActive = this.props.isActive || section.disableAccordions`

Reply:
Oh dang! 😁 Babysteppin'…